src/pages/spawn/spawnHost/SpawnHostButton.tsx

```diff
@@ -27,7 +27,7 @@ export const SpawnHostButton = () => {
     GetSpruceConfigQuery,
     GetSpruceConfigQueryVariables
   >(GET_SPRUCE_CONFIG);
-  const { search } = useLocation<{ search: string }>();
+  const { search } = useLocation();
```

Comment:
Why remove the typing?

Reply:
search is already defined in the location type declaration: https://github.com/remix-run/history/blob/main/docs/api-reference.md#location